CONTRIBUTING.md

```diff
@@ -1,3 +1,58 @@
+# Contributing
+
+## Getting started
+
 The Contributing page in the wiki provides some guidance on how to get started developing Uyuni.
 
 https://github.com/uyuni-project/uyuni/wiki/Contributing
+
+## Pull Requests
+
+The following roles exist in the context of a Pull Request:
+
+- Creator
+- Reviewer
+- Collaborators
+
+The guidelines noted here describe the expectations that should be met when submitting a Pull request.
+
+### Pull Request Creators
+
+1. Please fill out the Pull Request template.
+2. Please provide a description for someone not familiar with your code changes to be still able to understand your changes in the context of the project.
+3. Please enable the reviewer with the provided information in the description to understand the PR without opening any external resources like bug reports.
+4. Please ensure that before hitting the "Create" button on GitHub you have read and understood the `CONTRIBUTING.md` and linked GitHub Wiki pages.
+5. Please ensure all automation on GitHub receives a positive result checkmark.
+6. Please state in a comment why a negative result - if occurring - is incorrect or not your fault.
+7. Please use the GitHub functionality of re-requesting a review in case you haven't gotten a review after a reasonable time.
+8. Please re-request a review in case you made substantial changes to the Pull Request.
+9. Please write detailed and meaningful commit messages. Avoid the usage of generic messages like "fix bug". Further information can be found here:
```

Comment:
Nitpick: It's not exactly a sequential list of instructions so IMHO an unordered list would be more appropriate.

```suggestion
- Please fill out the Pull Request template.
- Please provide a description for someone not familiar with your code changes to be still able to understand your changes in the context of the project.
- Please enable the reviewer with the provided information in the description to understand the PR without opening any external resources like bug reports.
- Please ensure that before hitting the "Create" button on GitHub you have read and understood the `CONTRIBUTING.md` and linked GitHub Wiki pages.
- Please ensure all automation on GitHub receives a positive result checkmark.
- Please state in a comment why a negative result - if occurring - is incorrect or not your fault.
- Please use the GitHub functionality of re-requesting a review in case you haven't gotten a review after a reasonable time.
- Please re-request a review in case you made substantial changes to the Pull Request.
- Please write detailed and meaningful commit messages. Avoid the usage of generic messages like "fix bug". Further information can be found here:
```